mithril-aggregator/src/tools/signer_importer.rs

```diff
@@ -84,10 +95,6 @@ pub trait SignersImporterPersister: Sync + Send {
 #[async_trait]
 impl SignersImporterPersister for SignerStore {
     async fn persist(&self, signers: HashMap<PartyId, Option<PoolTicker>>) -> StdResult<()> {
-        info!(
```

Comment:
Is it wanted to remove this ?

Reply:
It has not been removed but displaced to where it was called, in the `run` method above,  since I don't have access to the logger here.